db/migrate/20210514165028_add_new_column_to_cards.rb

```diff
@@ -0,0 +1,5 @@
+class AddNewColumnToCards < ActiveRecord::Migration[6.1]
```

Comment:
Add alpha2Code (country code) to query the weather using city name and country code